airflow/config_templates/config.yml

```
@@ -1351,6 +1351,20 @@
       example: "/path/to/my_html_content_template_file"
       default: ~
       see_also: ":doc:`Email Configuration </howto/email-config>`"
+    - name: email_from_email
+      description: |
+        Email address that will be used as sender address.
+      version_added: 2.2.0
+      type: string
+      example: "airflow@example.com"
+      default: ~
+    - name: email_from_name
```

This does not match airflow.cfg

```suggestion
    - name: from_email
      description: |
        Email address that will be used as sender address.
      version_added: 2.2.0
      type: string
      example: "airflow@example.com"
      default: ~
    - name: from_name
```

---

Changed the options in default_airflow.cfg to match config.yml.

---

Also, as I mentioned in the other PR, `from_name` is not needed at all since `from_email` can just be set to `Display Name <mail@example.com>`.

---

> Also, as I mentioned in the other PR, `from_name` is not needed at all since `from_email` can just be set to `Display Name <mail@example.com>`.

It would have to be split for Sendgrid implementation because it expects two separate values.

---

It doesn’t, Sendgrid accepts the `Display <email>` form natively, the `Email(addr, name)` format is actually for compatibility.

https://github.com/sendgrid/sendgrid-python/blob/main/sendgrid/helpers/mail/email.py#L16-L40

---

Okay, will refactor

---

I don't want to be picky but I don't think `email_from_email` is a good configuration option name, why not `from_email` or `sender_email`

---

I agree, I was initially following one of the other PR's and inherited the naming from it